Routes/admin/index.js

```diff
@@ -70,6 +121,24 @@ adminRouter.get('/films', (req, res) => {
   });
 });
 
+adminRouter.delete('/films/:id', (req, res) => {
+  Film.findByIdAndDelete(req.params.id, (err, data) => {
```

Comment:
The same questions as with users. What will be with sessions for this film? Do we really need to delete films?